aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -401,5 +435,19 @@ TORCH_IMPL_FUNC(_convert_indices_from_coo_to_csr_structured_cpu) (
   }
 }
 
+TORCH_IMPL_FUNC(_convert_indices_from_csr_to_coo_structured_cpu) (
+  const Tensor& crow_indices, const Tensor& col_indices, const int64_t size, const bool out_int32, const Tensor& result
+) {
+  if (out_int32) {
+    AT_DISPATCH_INTEGRAL_TYPES(crow_indices.scalar_type(), "convert_indices_from_csr_to_coo_cpu", [&] {
+      convert_indices_from_csr_to_coo_cpu<scalar_t, int>(result, crow_indices, col_indices, size);
```

Comment:
Nit:
```suggestion
      convert_indices_from_csr_to_coo_cpu<scalar_t, int32_t>(result, crow_indices, col_indices, size);
```

Reply:
Done. It's not really consistent with the nearby code, but it seems like the better option.